components/brave_sync/brave_sync_service_unittest.cc

```diff
@@ -246,6 +250,37 @@ class BraveSyncServiceTest : public testing::Test {
   base::ScopedTempDir temp_dir_;
 };
 
+class BraveSyncServiceTestDelayedLoadModel : public BraveSyncServiceTest {
+ public:
+  void PrepareModelFactory() override {
```

Comment:
nit: I think `SetBookmarkModelFactory` would make more sense here

Reply:
I was hesitant between `Set` and `Prepare`, so will change to Set.

Reply:
this was fixed at https://github.com/brave/brave-core/pull/4588/files#diff-8b38d039014ebfe5a6c917bf6ee86992R216